### ISSUE TITLE
Provide alternate authentication mechanism for local development

### ISSUE THREAD

**Issue by upkarlidder** [Call-for-Code-for-Racial-Justice/TakeTwo-WebAPI#15]
**Is your feature request related to a problem? Please describe.**
It is hard for developers to code locally with the dependency on App Id

**Describe the solution you'd like**
Provide a way to either disable authentication or provide an alternate way to authentication when developing locally.

**Describe alternatives you've considered**
- refactor the authentication piece outside the main FastAPI class
- use configuration to load AppID class or local dev class

**Comment by josiemundi** [comment]
Yes @upkarlidder this one is important. I will assign this to @sebastiaanzaaijer. He said he will work on this :)

**Comment by josiemundi** [comment]
Reopening for Sprint 1

**Comment by sebastiaanzaaijer** [comment]
I'm working on a concept for this based on [this](https://fastapi.tiangolo.com/tutorial/security/oauth2-jwt/) example in the FastAPI docs to also allow for local logins. Hope to make a pull request in the next few days.

**Comment by sebastiaanzaaijer** [comment]
I've not really made any progress since my last comment. You can see where I've got to [here](https://github.com/sebastiaanzaaijer/TakeTwo-WebAPI/tree/15-alternative-login-methods). Unfortunately it's a bit too basic for a pull request and I'm not sure I'll get much time to work on this in the next weeks. Happy for someone else to progress this in the mean time, if not I'll try and pick it up again as soon as time allows.
